Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -1008,6 +1003,128 @@
+        <SoundArray index="Ready" value="NydusWorm_Ready"/>
+        <SoundArray index="What" value="NydusWorm_What"/>
+        <SoundArray index="Yes" value="NydusWorm_What"/>
+        <SoundArray index="Construction" value="Zerg_RockBuildingMorph_What"/>
+        <UnitIcon value="Assets\Textures\btn-building-zerg-nydusworm.dds"/>
+        <Wireframe>
+            <Image value="Assets\Textures\Wireframe-Zerg-NydusWorm.dds"/>
+        </Wireframe>
+    </CActorUnit>
+    <CActorUnit id="AP_NydusNetwork" parent="ZergBuildingEx" unitName="AP_NydusNetwork">
+        <Aliases value="_UnitHuge"/>
+        <On Terms="AbilTransport.AP_NydusCanalTransport.TransportLoad" Send="$Spell"/>
+        <On Terms="AbilTransport.AP_NydusCanalTransport.TransportUnload" Send="$Spell"/>
+        <On Terms="UnitDeathCustomize; ValidateUnit AP_HasZergGenericDeath" Send="DeathCustomize Detached"/>
+        <Model value="NydusNetwork"/>
+        <BuildModel value="NydusNetwork"/>
+        <DeathArray index="Normal" ModelLink="NydusNetworkDeath" SoundLink="NydusNetwork_Explode"/>
+        <DeathArray index="UnderConstruction" ModelLink="NydusNetworkDeathUnderConstruction" SoundLink="NydusNetwork_Explode"/>
+        <PlacementModel value="NydusNetworkPlacement"/>
+        <PortraitModel value="NydusNetworkPortrait"/>
+        <BarOffset value="90"/>
+        <BarWidth value="135"/>
+        <ErrorArray Error="CantSeeBuildLocation" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <ErrorArray Error="CantSeeLandLocation" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <ErrorArray Error="CantBuildLocationInvalid" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <ErrorArray Error="CantLandLocationInvalid" Sound="Alert_ZergBuildError" GroupSound="Alert_ZergBuildError"/>
+        <GroupIcon>
+            <Image value="Assets\Textures\Wireframe-Zerg-NydusNetwork.dds"/>
+        </GroupIcon>
+        <HeroIcon value="Assets\Textures\btn-building-zerg-nydusnetwork.dds"/>
+        <SoundArray index="Ready" value="NydusNetwork_Ready"/>
+        <SoundArray index="What" value="NydusNetwork_What"/>
+        <UnitIcon value="Assets\Textures\btn-building-zerg-nydusnetwork.dds"/>
+        <Wireframe>
+            <Image value="Assets\Textures\Wireframe-Zerg-NydusNetwork.dds"/>
+        </Wireframe>
+    </CActorUnit>
+    <CActorModel id="AP_NydusNetworkBuild" parent="ZergBuildingBuild">
+        <Model value="NydusNetworkBuild"/>
+    </CActorModel>
+    <CActorSimple id="AP_NydusCanalBuild">
+        <On Terms="ActorCreation" Send="Destroy"/>
+    </CActorSimple>
+    <!--                                                                                         -->
+    <!-- Nydus Network ......................................................................... -->
+    <!--                                                                                         -->
+    <CActorUnit id="NydusNetwork">

```

Comment:
This actor unit lacks unit and parent attributes.

Reply:
Checking again, these seem to be overrides? Need to double-check that these actors exist in liberty.sc2mod, and probably annotate them with a `<!-- Override -->` to explain what they're doing (or make AP copies and not use the old ones, though that's more verbose. I think `NydusNetwork` at least is unnecessary.)